### ISSUE TITLE
vertical rangeslider not filling up container

### ISSUE THREAD

**Issue by bsdis** [palantir/blueprint#4264]
I am using the rangeslider vertically, but for some reason it does not fill up its entire container. How do i set something like height:100% or fill={true}?
Example of the problem can be seen here:

https://codesandbox.io/s/weathered-breeze-ytnl4?file=/src/App.tsx

**Comment by adidahiya** [comment]
This is not a supported feature of `Slider` / `RangeSlider`. These components take some DOM measurements to implement their layout. Adding a prop like `fill: boolean` would be a new feature request.

**Comment by bsdis** [comment]
I see, ok... It actually seems to work when just setting 
```
.bp3-vertical {
  height: 100% !important;
}
```

Is it somehow possible (hack) to reverse the numbers so that it goes from max to min instead of min to max?

**Comment by adidahiya** [comment]
No, that's an open FR: https://github.com/palantir/blueprint/issues/4142
